tests/functional_tests/test_groups/models/ernie_vl/ernie45_vl_vit_debug.py

```
@@ -0,0 +1,356 @@
+        # HF RoPE
+        hf_rotary = hf_vit.rot_pos_emb(grid_thw)  # [N, 40]
+        hf_emb = torch.cat((hf_rotary, hf_rotary), dim=-1)  # [N, 80]
+        hf_cos = hf_emb.cos()
+        hf_sin = hf_emb.sin()
+
+        # MG RoPE
+        mg_rotary = mg_vit.rot_pos_emb(grid_thw)  # [N, 40]
+        mg_emb = torch.cat((mg_rotary.reshape(total_patches, 1, 1, -1),
+                           mg_rotary.reshape(total_patches, 1, 1, -1)), dim=-1)
+        mg_cos = mg_emb.cos().flatten()
+        mg_sin = mg_emb.sin().flatten()
```

Flake8 (7.3.0)

[error] local variable 'mg_sin' is assigned to but never used

(F841)